google/resource_dataproc_cluster.go

```diff
@@ -797,7 +797,7 @@ func flattenPreemptibleInstanceGroupConfig(d *schema.ResourceData, icg *dataproc
 func flattenInstanceGroupConfig(d *schema.ResourceData, icg *dataproc.InstanceGroupConfig) []map[string]interface{} {
 	disk := map[string]interface{}{}
 	data := map[string]interface{}{
-	//"instance_names": []string{},
+		//"instance_names": []string{},
```

Comment:
this can be reverted, it's not really relevant here

Reply:
Reverted.